parm/atm/obs/config/sondes.yaml

```diff
@@ -0,0 +1,309 @@
+obs space:
+  name: sondes
+  obsdatain:
+    obsfile: $(OBS_DIR)/$(OBS_PREFIX).sondes.$(OBS_DATE).nc4
+    obsgrouping:
+      group variables: ["station_id", "LaunchTime"]
+      sort variable: "air_pressure"
+      sort order: "descending"
+  obsdataout:
+    obsfile: $(DIAG_DIR)/diag_sondes_$(OBS_DATE).nc4
+  simulated variables: [eastward_wind, northward_wind, air_temperature, specific_humidity, surface_pressure]
+obs operator:
+  name: Composite
+  components:
+   - name: VertInterp
+     variables:
+     - name: air_temperature
+#    - name: virtual_temperature
+     - name: specific_humidity
+     - name: eastward_wind
+     - name: northward_wind
+   - name: SfcPCorrected
+     variables:
+     - name: surface_pressure
+     da_psfc_scheme: UKMO
+#
```

Comment:
Tests with ufs_da added the prior to the obs filters section
```
linear obs operator:
  name: Composite
  components:
   - name: VertInterp
     variables:
     - name: air_temperature
     - name: specific_humidity
     - name: eastward_wind
     - name: northward_wind
   - name: Identity
     variables:
     - name: surface_pressure
```
Not sure if this is still needed.

Reply:
Yes, thanks for this and the other. This is a great example of why, going forward, it will be much better for us to maintain our own YAMLs and not rely on the EWOK ones.